common/src/main/java/com/wynntils/models/players/PlayerModel.java

```diff
@@ -31,24 +33,39 @@
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
 public final class PlayerModel extends Model {
+    private static final String ATHENA_USER_NOT_FOUND = "User not found";
     private static final Pattern GHOST_WORLD_PATTERN = Pattern.compile("^_(\\d+)$");
+
+    // If there is a failure with the API, give it time to recover
+    private static final int ERROR_TIMEOUT_MINUTE = 5;
+
+    // Max amount of overall lookup errors
     private static final int MAX_ERRORS = 5;
 
+    // Max amount of errors for a single user,
+    // before disabling lookups for them
+    private static final int MAX_USER_ERRORS = 3;
+
     private final Map<UUID, WynntilsUser> users = new ConcurrentHashMap<>();
+    private final Set<UUID> usersWithoutWynntilsAccount = ConcurrentHashMap.newKeySet();
     private final Set<UUID> fetching = ConcurrentHashMap.newKeySet();
     private final Map<UUID, Integer> ghosts = new ConcurrentHashMap<>();
-    private int errorCount;
     private final Map<UUID, String> nameMap = new ConcurrentHashMap<>();
 
+    // Count of errors in the last minute, to avoid spamming the API
```

Comment:
```suggestion
    // The size of this set is the count of errors in the last ERROR_TIMEOUT_MINUTE minutes.
    // This is used to avoid spamming the API.
```